storybook/entry.js

```diff
@@ -11,7 +11,9 @@ import fetchMixin from '../mixins/fetch.client'
  * @nuxtjs/storybook
  * execute plugin scripts
  */
-window.__NUXT__ = window.__NUXT__ || {}
+window.__NUXT__ = window.__NUXT__ || {
```

Comment:
What is the case of `window.__NUXT__` having initial value? Should we also apply config for that?

Reply:
🤔
Before this line `window.__NUXT__` always `undefined`. It could be a simple assignment

Reply:
So why we haver `window.__NUXT__ ||` in first place if is always `undefined`? :D I mean something is either extra or undocumented.

Reply:
I was trying to inject `__NUXT__` definition into HTML, instead of entry. But after struggling with HTML thing I changed my mind and I must forget to cleanup this line. 😅